### DISCUSSION TITLE
A doubt about the time series chain

### DISCUSSION THREAD

**Discussion by adityabhandwalkar** [TDAmeritrade/stumpy#807] [General]
Hello Community, 

I have a general query about the time series chain and general motif search:

1. With time series chain , as we advance further into the time series data, patterns do evolve in significant manner and they tend to elaborate the process in terms of change happening over the course of time. While using  the `allc` we compute the unanchored chain, My question is during the process when we opt for the different regions of chains and they have the different colors and of different size , is it possible to label them based on feature of size and categorized them in terms of patterns which are `normal` , `not so normal` and `abnormal` . Secondly while other region in time series is not colored and remains same does this mean that the only colored region seems to have evolution and rest of the data is normal? as shown in figure

![chain](https://user-images.githubusercontent.com/56731860/223395301-9e456496-ecca-48bc-a347-45ec35edb3a7.png)

2. With motif search a can we rank them as in   `normal` , `not so normal` and `abnormal`(try to plot them for e.g in Red , yellow or orange according to their normality) just for the clarification of the extracted motif patterns or it is completely useless to classify the motif within the categories. 

@seanlaw  Do you have any thoughts/remarks about this  ?

**Comment by seanlaw** [comment]
> @seanlaw Do you have any thoughts/remarks about this ?

Sorry, I do not have much experience with time series chains.

**Reply by adityabhandwalkar** [reply]
Do you have any suggestion on point number 2 , for classifying the motifs?

**Reply by NimaSarajpoor** [reply]
I think your should take a look at the original paper, and then the tutorials in `stumpy/docs`.  (And, by take a look, I mean... read and understand it)
